### PR TITLE
Re-naming the EDRD Apex Classes & Triggers

### PR DESCRIPTION
**Pull request by deepakmulamalla**

### CODE REVIEW

File: force-app/main/default/triggers/MedicationRequestTrigger.trigger [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
-trigger EDRD_tgr_MedicationRequest on MedicationRequest (before insert, before Update, after Update) {
+trigger MedicationRequestTrigger on MedicationRequest (before insert, before Update, after Update) {
     
     List<MedicationRequest> MRListValidate = new List<MedicationRequest>();
     
     if(trigger.isBefore){
         if(trigger.isInsert || trigger.isUpdate){
             for(MedicationRequest MRObj : trigger.new){
+               /* if(MRObj.Dosage_Units_Name__c != MRObj.Strength_Units__c){
+                    MRObj.addError(label.EDRD_label_DosageAndStrengthUnits);
```
Comment (ckleinacn) [inline force-app/main/default/triggers/MedicationRequestTrigger.trigger:18]: @deepakmulamalla - Could the commented lines of code be removed?

Comment (deepakmulamalla) [reply force-app/main/default/triggers/MedicationRequestTrigger.trigger]: Done @ckleinacn, Thanks for letting me know!

**Review by cgijeffolsen — APPROVED** [review]
Thank you for your efforts on this @deepakmulamalla @NataliaNikishina

**Review by RanadheerRG — APPROVED** [review]
@deepakmulamalla Approved

**Comment by deepakmulamalla** [comment]
Thanks for approving @cgijeffolsen

**Pull request merged**